### PR TITLE
fix: adds babel/polyfill to fix regeneratorRuntime

### PR DESCRIPTION
**Pull request by rmathis**

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
Does this work with other components on the page, we've run into issues with Babel polyfill before.
uPortal web components ended up solving the issue this way, so everything is scoped.
https://github.com/uPortal-contrib/uPortal-web-components/pull/44

File: notification-portlet-webcomponents/notification-modal/src/main.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import "@babel/polyfill";
+import 'promise-polyfill/src/polyfill';
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-modal/src/main.js]: Web Components present many of the same challanges as libraries, when loaded on a page.

---

> This is useful it you are building a website and want to support older browsers. Javascript library authors should NOT use this method.

```
import 'promise-polyfill/src/polyfill';
```

> If you would like to not affect the global environment (sometimes known as a ponyfill), you can import the base module. This is nice for library authors or people working in environment where you don't want to affect the global environment.

```
import Promise from 'promise-polyfill';
```

Source: https://github.com/taylorhakes/promise-polyfill

---

The Ponyfill looks like it should scope things and avoid polyfill conflicts.

Comment (rmathis) [reply notification-portlet-webcomponents/notification-modal/src/main.js]: This causes it to break, regardless of where I put it.

Comment (ChristianMurphy) [reply notification-portlet-webcomponents/notification-modal/src/main.js]: 🤔 what about leveraging http://bluebirdjs.com/docs/getting-started.html as the promise ponyfill?

File: notification-portlet-webcomponents/notification-modal/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "vue-cli-service serve --port=8081",
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-modal/package.json]: Is the port customization still needed?

**Comment by drewwills** [comment]
Are you guys pretty happy with this PR at this point?  Is this the last of what we need in Notification?

**Comment by ChristianMurphy** [comment]
I'm happy with it, it can be merged as is.
This is the last of the JavaScript polyfills, the last item is a WebComponent polyfill fix (https://github.com/uPortal-contrib/uPortal-web-components/issues/86) coming in a separate PR.

**Pull request merged**